### PR TITLE
fix: set keepAlive false to parse webview pages correctly Node.js >=19

### PR DESCRIPTION
**Pull request by yaumu3**
Fix for https://github.com/appium/appium/issues/19678

### CODE REVIEW

File: lib/helpers/webview.ts [hunk 1]
```diff
@@ -323,8 +324,8 @@ async function collectWebviewsDetails(
   for (const item of webviewsMapping) {
     detailCollectors.push(
       (async () => {
-        let port: number|undefined;
-        let host: string|undefined;
+        let port: number | undefined;
+        let host: string | undefined;
```
Comment (KazuCocoa) [inline lib/helpers/webview.ts:328]: by auto-format or something?

**Review by KazuCocoa — APPROVED** [review]

File: lib/helpers/webview.ts (hunk 1 above)
Comment (yaumu3) [reply lib/helpers/webview.ts:328]: Yes, by the pre-commit hook triggered via husky.

File: lib/helpers/webview.ts [hunk 2]
```diff
@@ -357,6 +358,11 @@ async function cdpList(host: string, port: number): Promise<object[]> {
     await axios({
```
Comment (mykola-mokhnach) [inline lib/helpers/webview.ts:358]: this API can be extracted into a separate function call to avoid extra duplication

Comment (yaumu3) [reply lib/helpers/webview.ts:358]: Thank you! Extracted as 3e1ea66eee409e56519e5d1f885ec048dc66c1fe

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**